src/core/maps/const_occupancy_estimator.h

```diff
@@ -1,12 +1,23 @@
+
 #ifndef __CONST_OCCUPANCY_ESTIMATOR_H
 #define __CONST_OCCUPANCY_ESTIMATOR_H
 
 #include "cell_occupancy_estimator.h"
 
+/**
+ * \brief This class is responsible for strategy of base occupancy estimation.
+ */
+
 class ConstOccupancyEstimator : public CellOccupancyEstimator {
 public:
   ConstOccupancyEstimator(double occ, double empty) :
     CellOccupancyEstimator(occ, empty) {}
+
+/**
+ * This virtual method returns base probability that cell is occupied;
+ * ignores fist two parameters cause in this class implemented strategy of base occupancy estimation.
+ */
+
```

Comment:
1. Remove the empty line below the comment.
2. Clean up suggestions: "Returns base probability <...> uses only the cell status returned by a sensor for estimation. "